src/challonge.service.js

```diff
@@ -53,6 +49,30 @@ const fetchMembers = ({ api, organization }) => async function () {
     )(tournamentsDetails);
 };
 
+const fetchOpenTournamentsForMember = ({ api, organization }) => async function (memberEmailHash, { signedUp = undefined, includeUnderway = true } = {}) {
+    const tournaments = await fetchOpenTournaments({ api, organization })(includeUnderway);
+
+    const tournamentsDetailsPromises = R.pipe(
+        R.map(({ id }) => id),
+        R.map(fetchTournament({ api })),
+    )(tournaments);
+    const tournamentsDetails = await Promise.all(tournamentsDetailsPromises);
+
+    const filterFunc = R.cond([
```

Comment:
Is R.cond really necessary? Can't this just be a simpler lamda?

Reply:
You're right, it's a bit overkill. I just wanted to express  the tristatedness in a cleaner way than nested ternary operators. But something like this should work:
```
const filterFunc = R.isNil(signedUp)
    ? R.T
    : T.equal(signedUp)
```

What dou you think? :)

(And `signedUp` should probably be renamed to `signedUpFilter`.)

Reply:
Updated now!

Reply:
Looks better!